docs/tutorials/kuai-cli-deploy-contract.md

```diff
@@ -0,0 +1,158 @@
+Rewrite Cargo.toml
+Rewrite capsule.toml
+Done
+```
+
+2. Build contract
+
+- `kuai contract build --name [contract-name] --release`
+
+```bash
+$ kuai contract build --name always-success --release
+
+Building contract always-success
+   Compiling always-success v0.1.0 (/code/contracts/always-success)
+Done
+```
+
+3. Deploy contract
+
+- `kuai contract deploy --name [contract-name] --from [0x.....] --signer ckb-cli`
+
+```bash
+$ kuai contract deploy --name always-success --from ckt1qyqw8yx5hx6vwcm7eqren0d0v39wvfwdhy3q2807pp --signer ckb-cli
+
+[warn] ANYONE_CAN_PAY script info not found in config!
+[warn] OMNILOCK script info not found in config!
+[warn] `config` changed, regenerate lockScriptInfos!
+The address format generated by generateAddress or scriptToAddress will be deprecated, please migrate to encodeToAddress to generate the new ckb2021 full format address as soon as possible
+Input ckt1qzda0cr08m85hc8jlnfp3zer7xulejywt49kt2rr0vthywaa50xwsq0rjr2tndx8vdlvspuehkhkgjhxyhxmjgsqw8hsk's password for sign messge by ckb-cli:
+deploy success, txHash:  0x7507b59b5f715c0fcd83d55df5569d72ff1cb05cd399afe1f366e03918827a58
+```
+
+## deploy contract workflow by export tx:
+
+1. `kuai contract deploy --name [contract-name] --from [0x.....] --export [transaction json file]`
+
+```bash
+$ kuai contract deploy --name always-success --from ckt1qyqw8yx5hx6vwcm7eqren0d0v39wvfwdhy3q2807pp --export ./tx.json
+
+[warn] ANYONE_CAN_PAY script info not found in config!
+[warn] OMNILOCK script info not found in config!
+[warn] `config` changed, regenerate lockScriptInfos!
+```
+
+2. `ckb-cli tx sign-inputs --tx-file [transaction json file] --from-account 0xe390d4b9b4c7637ec80799bdaf644ae625cdb922 --add-signatures`
+
+```bash
+$ ckb-cli tx sign-inputs --tx-file ./tx.json --from-account 0xe390d4b9b4c7637ec80799bdaf644ae625cdb922 --add-signatures
+
+Password:
+- lock-arg: 0xe390d4b9b4c7637ec80799bdaf644ae625cdb922
+  signature: 0xf3af55aa4c9d6923852465a33d625ff2b5489409908259d79d6aa5bf15a407f10ec32cdd9a9ebff837cdccc2850fc533cab8f08a4c701b57e75530c329f56be201
+```
+
+3. `ckb-cli tx send --tx-file [transaction json file]`
+
+```bash
+$ ckb-cli tx send --tx-file ./tx.json
+
+0xfd79d9c3e562b2c5c51df39e49fbb906c63cf2e33e5dbe2bbff12345fa4e6c72
+```
+
+## export tx workflow with multisig
+
+1. `kuai contract deploy --name [contract-name] --from multisig [R] [M] [...args] --export [transaction json file]`
+
+```bash
+$ kuai contract deploy --name always-success  --from multisig 0 2 0xe390d4b9b4c7637ec80799bdaf644ae625cdb922 0xb6ddba87bb5af5f053b1ae5bcbc7f4de03479f7e --export ./tx.json
+
+[warn] ANYONE_CAN_PAY script info not found in config!
+[warn] OMNILOCK script info not found in config!
+[warn] `config` changed, regenerate lockScriptInfos!
+The address format generated by generateAddress or scriptToAddress will be deprecated, please migrate to encodeToAddress to generate the new ckb2021 full format address as soon as possible
+```
```

Comment:
```suggestion
  ```bash
  # $ kuai contract deploy --name always-success  --from multisig 0 2 0xe390d4b9b4c7637ec80799bdaf644ae625cdb922 0xb6ddba87bb5af5f053b1ae5bcbc7f4de03479f7e --export ./tx.json

  # [warn] ANYONE_CAN_PAY script info not found in config!
  # [warn] OMNILOCK script info not found in config!
  # [warn] `config` changed, regenerate lockScriptInfos!
  # The address format generated by generateAddress or scriptToAddress will be deprecated, please migrate to encodeToAddress to generate the new ckb2021 full format address as soon as possible
  ```
```